hamilton/plugins/h_ddog.py

```diff
@@ -0,0 +1,259 @@
+from typing import Any, Dict, Optional
+
+from ddtrace import Span, context, tracer
+
+from hamilton import lifecycle
+
+
+class DDOGTracer(
```

Comment:
Could add testing but I don't think its worth it, I'll just fix forward and if its a pain add testing